src/app/register/allPages/EmailPassword.tsx

```diff
@@ -0,0 +1,23 @@
+import GoBackArrow from "../_components/GoBackArrow";
+import ComponentsGround from "../_components/EmailPasswordComponentsGround";
+import { type User } from "../interfaces";
+
+interface EmailPasswordProps {
+  setData: (data: User) => void;
+  setPage: (page: string) => void;
+  page: string;
+  data: User;
+}
+
+export default function EmailPassword(props: EmailPasswordProps) {
+  return (
+    <div className="h-[100%] min-h-[100vh] w-[100%] min-w-[100vw] overflow-hidden bg-primary-50 p-6 md:p-9">
```

Comment:
`h-[100%]` is equal to `h-full`, same for `w-[100%]`
`min-h-[100vh]` is equal to `min-h-screen`, same for min-w